integration-cli/docker_cli_nat_test.go

```diff
@@ -105,7 +92,5 @@ func (s *DockerSuite) TestNetworkLoopbackNat(c *check.C) {
 	out, _ := dockerCmd(c, "run", "-t", "--net=container:server", "busybox",
 		"sh", "-c", fmt.Sprintf("stty raw && nc -w 5 %s 8080", endpoint.String()))
 	final := strings.TrimRight(string(out), "\n")
-	if final != msg {
-		c.Fatalf("Expected message %q but received %q", msg, final)
-	}
+	c.Assert(final, check.Equals, msg, check.Commentf("Expected message %q but received %q", msg, final))
```

Comment:
The error comment is nothing special so I think `c.Assert(final, checker.Equals, msg)` is OK.